ext/build_defaults.yaml

```diff
@@ -21,6 +21,9 @@ build_msi:
   facter:
     ref: 'refs/tags/2.3.0'
     repo: 'git://github.com/puppetlabs/facter.git'
+  cfacter:
+    ref: 'cfacter-553c5dd5eaeda6ee6c8591ab536e776390c58f8b.zip'
+    repo: 'http://builds.puppetlabs.lan/cfacter/553c5dd5eaeda6ee6c8591ab536e776390c58f8b/artifacts/windows/cfacter-553c5dd5eaeda6ee6c8591ab536e776390c58f8b.zip'
```

Comment:
Yes, that is very true. I'll look into what I can do to change this. I do like having the name and url separate, so I'll probably maintain that, and remove the package name from the end of the path